### PR TITLE
#30510 now you can add wf comments

### LINKED ISSUE #30510: Implement Activities API Endpoints for Edit Contentlet
### Parent Issue
#30372 

### User Story
As a developer, I want to create the necessary API endpoints for Activities management in the Edit Contentlet Sidebar, so we can handle all activities-related operations efficiently.

### Acceptance Criteria
1. API Implementation
   - Create GET endpoint for listing activities with pagination
   - Implement POST endpoint for adding new activities
   - Create DELETE endpoint for removing activities
   - Include proper error handling and status codes
   - Add proper validation for inputs

CleanShot 2024-10-30 at 13 12 01@2x

2. Response Structure
To be defined

3. Documentation
   - OpenAPI/Swagger documentation
   - Example requests and responses

### Proposed Objective
Core Features

### Proposed Priority
Priority 2 - Important

### PR DESCRIPTION
**Pull request by jdotcms**
Now there is an endpoint to add wf comments

This PR fixes: #30510

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java [hunk 1]
```diff
@@ -5700,4 +5700,85 @@ private WorkflowTimelineItemView toWorkflowTimelineItemView(final WorkflowTimeli
...
 
+    /**
+     * Creates a new workflow comment
+     *
+     * @param request HttpServletRequest
+     * @param workflowSchemeForm WorkflowSchemeForm
+     * @return Response
+     */
+    @POST
+    @Path("/comments/{contentletId}")
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java:5711]: I think this contentletId should be in the form

Comment (jdotcms) [reply dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java:5708]: why the contentlet is the owner of the comment generated, the form will be the detail of it
I would guess it make some kinda sense for me

Comment (freddyDOTCMS) [reply dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java:5708]: If the Contetlet if the owner then should it be "{contentletId}/comments"?

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java [hunk 2]
```diff
@@ -5697,4 +5697,85 @@ private WorkflowTimelineItemView toWorkflowTimelineItemView(final WorkflowTimeli
...
+            description = "Create a [workflow comment].\n\n " +
+                    "Returns created workflow comment on success.",
+            tags = {"Workflow"},
+            responses = {
+                    @ApiResponse(responseCode = "200", description = "Copied workflow comment successfully",
+                            content = @Content(mediaType = "application/json",
+                                    schema = @Schema(implementation = ResponseEntityWorkflowCommentView.class)
+                            )
+                    ),
+                    @ApiResponse(responseCode = "400", description = "Bad request"), // invalid param string like `\`
```
Comment (freddyDOTCMS) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java:5723]: I don't know if we have all these cases tested

**Review by freddyDOTCMS — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

**Pull request merged**